### ISSUE TITLE
transformSchema removes directives

### ISSUE THREAD

**Issue by kbrandwijk** [prisma/graphql-transform-schema#5]
See https://github.com/apollographql/apollo-server/issues/643 for more details.

**Comment by kbrandwijk** [comment]
I think this might be caused by the fact that a new schema is created, instead of manipulating the schema that is passed in. https://github.com/graphcool/graphql-transform-schema/blob/master/src/index.ts#L18

**Comment by schickling** [comment]
Good catch @kbrandwijk. Would you mind creating a PR that additionally copies the directives as well?

**Comment by kbrandwijk** [comment]
@schickling I think it's a bit more than that, because if you specify any custom directive in your schema, it doesn't automatically show up in the `schema.getDirectives()` result.

I think the necessary refactoring is to not create a new schema, but I'm not 100% sure how.
